controller/input.h

```diff
@@ -27,6 +27,8 @@
 #include <stdexcept>
 #include <linux/uinput.h>
 
+#define INPUT_MAX_FF_EFFECTS 10
```

Comment:
As already written, this should probably match with what ff-memless in the kernel supports.